deploy-manage/users-roles/cluster-or-deployment-auth/user-authentication.md

```diff
@@ -38,4 +43,37 @@ $$$oidc$$$
 
 $$$saml$$$
 
-$$$token-authentication$$$
\ No newline at end of file
+$$$token-authentication$$$
+
+
+
+Review the following topics to learn about authentication in your Elasticsearch cluster:
+
+### Set up user authentication
+
+* Learn about the available [realms](/deploy-manage/users-roles/cluster-or-deployment-auth/authentication-realms.md) that you can use to authenticate users
+* Manage passwords for [default users](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-users.md)
+* Manage users [natively](/deploy-manage/users-roles/cluster-or-deployment-auth/native.md)
+* Integrate with external authentication providers using [external realms](/deploy-manage/users-roles/cluster-or-deployment-auth/external-authentication.md):
+  * [Active Directory](/deploy-manage/users-roles/cluster-or-deployment-auth/active-directory.md)
+  * [JWT](/deploy-manage/users-roles/cluster-or-deployment-auth/jwt.md)
+  * [Kerberos](/deploy-manage/users-roles/cluster-or-deployment-auth/kerberos.md)
+  * [LDAP](/deploy-manage/users-roles/cluster-or-deployment-auth/ldap.md)
+  * [OpenID Connect](/deploy-manage/users-roles/cluster-or-deployment-auth/openid-connect.md)
+  * [SAML](/deploy-manage/users-roles/cluster-or-deployment-auth/saml.md)
+  * [PKI](/deploy-manage/users-roles/cluster-or-deployment-auth/pki.md)
+  * [Implement a custom realm](/deploy-manage/users-roles/cluster-or-deployment-auth/custom.md)
+* Configure [file-based authentication](/deploy-manage/users-roles/cluster-or-deployment-auth/file-based.md)
+* Enable [anonymous access](/deploy-manage/users-roles/cluster-or-deployment-auth/anonymous-access.md)
+* Manage [user profiles](/deploy-manage/users-roles/cluster-or-deployment-auth/user-profiles.md)
```

Comment:
I would suggest to put user profiles into the advanced topics.

Reply:
yeah... just reading the first paragraph I'd really move it :)

> The user profile feature is designed only for use by Kibana and Elastic’s Observability, Enterprise Search, and Elastic Security solutions. Individual users and external applications should not call this API directly

It's the type of low level document that probably ES devs created to disclose the information to Kibana and Solutions teams, but in general users don't really need to be aware of all that, and in case of needing they are probably unable to fix anything.